packaging/docker/Dockerfile.eks

```diff
@@ -31,6 +33,13 @@ RUN curl -sLO https://github.com/krallin/tini/releases/download/${TINI_VERSION}/
 COPY sidecar/requirements.txt /tmp
 RUN pip3 install -r /tmp/requirements.txt
 
+# Install flamegraph
+RUN curl -sLO https://raw.githubusercontent.com/brendangregg/FlameGraph/90533539b75400297092f973163b8a7b067c66d3/stackcollapse-perf.pl && \
+  curl -sLO https://raw.githubusercontent.com/brendangregg/FlameGraph/90533539b75400297092f973163b8a7b067c66d3/flamegraph.pl && \
+  sha256sum -c /tmp/flamegraph.sha256sum && \
```

Comment:
Maybe `echo` the content into `/tmp/flamegraph.sha256sum` so that the hashes are kept in this file together with URL. It'll be easier to maintain, IMHO.

Reply:
I've done it that way in the past, but sha256sum's parser for the sum file is whitespace sensitive.

Reply:
Look at the the [build Dockerfile(s)](https://github.com/apple/foundationdb/blob/master/build/docker/centos7/build/Dockerfile#L59) they keep the shasum string in the file to avoid maintaining the extra files and the COPY command.